### ISSUE TITLE
Inconsistent List-Parsing with code blocks

### ISSUE THREAD

**Issue by apfelbox** [erusev/parsedown#173]
Hi,

I noticed some incosistent list parsing Sometimes it creates a paragraph inside the `<li></li>` sometimes it doesn't. Seem to depend on the extra line:
### Without

``` md
* List
* List
* List with code
  ´´´
  this works!
  ´´´
* Test
```

Generates

``` html
<ul>
<li>List</li>
<li>List</li>
<li>List with code
<pre><code>this works!</code></pre></li>
<li>Test</li>
</ul>
```
### With Paragraph (notice the empty line)

``` md
* List
* List
* List with code
  ´´´
  this works!
  ´´´

* Test
```

Generates

``` html
<ul>
<li>List</li>
<li>List</li>
<li>
<p>List with code</p>
<pre><code>this works!</code></pre>
</li>
<li>Test</li>
</ul>
```

(I used `´` for the fenced code blocks in the examples because I can't get the github parse to not parse them in the examples)

**Comment by 33mhz** [comment]
This still appears to be an issue. Or maybe #176. It correctly wraps the contents of the last `<li>` with `<p>`, but not the first `<li>` in the list.

GitHub will turn
```


* one 
* two


* one
* two
* three
* four


* one


* one


```
into
```
<ul>
 <li>one</li>
 <li><p>two</p></li>
 <li><p>one</p></li>
 <li>two</li>
 <li>three</li>
 <li><p>four</p></li>
 <li><p>one</p></li>
 <li><p>one</p></li>
</ul>
```
Parsedown turns it into:
```
<ul>
 <li>one</li>
 <li><p>two</p></li>
 <li>one</li>
 <li>two</li>
 <li>three</li>
 <li><p>four</p></li>
 <li><p>one</p></li>
 <li>one</li>
</ul>
```

**Comment by aidantwoods** [comment]
I believe inconsistent addition of paragraphs should have been solved in https://github.com/erusev/parsedown/pull/475, which was included in Parsedown `1.7.1` so I'm going to close this for now, but let me know if there is an issue still present :)
